### PR TITLE
Taxi autonomous

### PR DESCRIPTION
**Pull request by kellytai**
completed command to drive a certain distance off the tarmac

### CODE REVIEW

File: codelabs/first_robot_code/src/main/java/com/gos/codelabs/first_code_lab/Robot.java [hunk 1]
```diff
@@ -64,7 +64,11 @@ public void teleopInit() {
     public void teleopPeriodic() {
         double speed = m_stick.getX();
         boolean trigger = m_stick.getTrigger(); // NOPMD
```
Comment (pjreiniger) [inline codelabs/first_robot_code/src/main/java/com/gos/codelabs/first_code_lab/Robot.java:66]: Doesn't seem like this should be part of this PR

File: codelabs/gitflow/src/main/java/com/gos/codelabs/gitflow/RobotContainer.java [hunk 2]
```diff
@@ -9,6 +9,7 @@
 import com.gos.codelabs.gitflow.subsystems.JacksonCodelab2020Part1Part1;
 import com.gos.codelabs.gitflow.subsystems.JaniseCodelab2021Pt1Subsystem;
 import com.gos.codelabs.gitflow.subsystems.JustinaCodelab2021Part1Subsystem;
+import com.gos.codelabs.gitflow.subsystems.KellyCodelab2021Part1Subsystem;
 import com.gos.codelabs.gitflow.subsystems.PJCodelab2021Part1Subsystem;
```
Comment (pjreiniger) [inline codelabs/gitflow/src/main/java/com/gos/codelabs/gitflow/RobotContainer.java:13]: Doesn't seem like this should be part of this PR

File: codelabs/gitflow/src/main/java/com/gos/codelabs/gitflow/RobotContainer.java [hunk 3]
```diff
@@ -40,7 +41,7 @@
     private AriellaCodelab2021Part1Subsystem m_ariellaCodelab2021Part1;
     private AmandaCodelab2021Part1 m_amandaCodeLab;
     private JustinaCodelab2021Part1Subsystem m_justinaCodeLab2021Part1;
-
+    private KellyCodelab2021Part1Subsystem m_kellyCodeLab2021Part1;
```
Comment (pjreiniger) [inline codelabs/gitflow/src/main/java/com/gos/codelabs/gitflow/RobotContainer.java:44]: Doesn't seem like this should be part of this PR

**Review by pjreiniger — APPROVED** [review]

**Comment by pjreiniger** [comment]
Don't worry if bazel fails the CI checks.

**Review by pjreiniger — CHANGES_REQUESTED** [review]
Changed my mind. Requesting changes

**Review by ashley-wei — COMMENTED** [review]
you did great!!! :)

**Review by pjreiniger — APPROVED** [review]

**Pull request merged**